hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ChunkKeyHandler.java

```diff
@@ -128,17 +131,23 @@ protected void execute(OzoneClient client, OzoneAddress address)
         chunkDetails.setChunkName(chunkInfo.getChunkName());
         chunkDetails.setChunkOffset(chunkInfo.getOffset());
         chunkDetailsList.add(chunkDetails);
-        chunkPaths.add(getChunkLocationPath(containerData.getContainerPath())
-              + File.separator
-              + chunkInfo.getChunkName());
+        if (chunkLayOutVersion == ChunkLayOutVersion.FILE_PER_CHUNK) {
+          chunkPaths.add(getChunkLocationPath(containerData.getContainerPath())
+                  + File.separator
+                  + chunkInfo.getChunkName());
+        } else {
```

Comment:
lets convert this if into a switch case, in case new CHunkLayoutVersions are added later, there should be an option of adding it later. Also for default case lets throw an exception.